src/main/java/page/nafuchoco/neojukepro/core/executor/RepeatCommand.java

```diff
@@ -57,12 +58,15 @@ public void onInvoke(CommandContext context) {
 
     @Override
     public String getDescription() {
-        return null;
+        return "Repeat the track.";
     }
 
     @Override
     public String getHelp() {
-        return null;
+        return getName() + "[option]/n----/n" +
+                "[NONE] Do not repeat./n" +
+                "[SINGLE] Repeat one song./n" +
+                "[ALL] Repeat all.";
```

Comment:
```suggestion
        return getName() + "[option]/n----/n" +
                "[NONE]: Do not repeat./n" +
                "[SINGLE]: Repeat one song./n" +
                "[ALL]: Repeat all.";
```
Add a semicolon between the argument and the description.